packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1086,6 +1181,23 @@ class FollowerLayer extends ContainerLayer {
 
   Offset _lastOffset;
   Matrix4 _lastTransform;
+  Matrix4 _invertedTransform;
+
+  @override
+  S find<S>(Offset regionOffset) {
+    if (link.leader == null && showWhenUnlinked) {
+      return super.find<S>(regionOffset - unlinkedOffset);
+    }
+    if (_invertedTransform == null) {
+      final Matrix4 transform = getLastTransform();
+      assert(transform != null);
+      _invertedTransform = new Matrix4.zero();
+      transform.copyInverse(_invertedTransform);
+    }
+    final Vector4 vector = new Vector4(regionOffset.dx, regionOffset.dy, 0.0, 1.0);
+    final Vector4 result = _invertedTransform.transform(vector);
+    return super.find<S>(new Offset(result[0] - linkedOffset.dx, result[1] - linkedOffset.dy));
```

Comment:
this doesn't seem to handle the case of `link.leader == null && !showWhenUnlinked`. I assume in that case we should just return null?

Reply:
Fixed